Dockerfile

```diff
@@ -12,7 +12,7 @@ FROM prepare AS development
 
 VOLUME /srv/app
 
-ENTRYPOINT ["docker-entrypoint.sh"]
+ENTRYPOINT ["/srv/app/docker-entrypoint.sh"]
```

Comment:
(following the latest sqitch update, this change seems to be necessary now, you might need to update your development image)

Reply:
You can resolve this comment once acknowledged